apps/epic-react/src/inngest/functions/discord/sync-discord-roles.ts

```diff
@@ -0,0 +1,134 @@
+import {inngest} from '@/inngest/inngest.server'
+import {DiscordError, DiscordMember} from '@/lib/discord'
+import {fetchAsDiscordBot, fetchJsonAsDiscordBot} from '@/lib/discord-query'
+import {
+  NEW_PURCHASE_CREATED_EVENT,
+  PURCHASE_STATUS_UPDATED_EVENT,
+} from '@skillrecordings/inngest'
+import {prisma} from '@skillrecordings/database'
+import {PurchaseStatusUpdatedEvent} from '@skillrecordings/inngest/src'
+
+// @ts-ignore
+BigInt.prototype.toJSON = function () {
+  return this.toString()
+}
+
+const EPIC_REACT_V1_PRODUCT_IDS = [
+  'kcd_910c9191-5a69-4019-ad1d-c55bea7e9714',
+  'kcd_8acc60f1-8c3f-4093-b20d-f60fc6e0cf61',
+  'kcd_2b4f4080-4ff1-45e7-b825-7d0fff266e38',
+]
+
+const EPIC_REACT_V2_PRODUCT_IDS = [
+  'kcd_product-clzlrf0g5000008jm0czdanmz',
+  'kcd_product_b394271c-d6d6-4403',
+  'kcd_product_15d22ad4-b668-4e81-bb5a',
+]
```

Comment:
would be better to look these up instead of hard coding, but here it is

feels bad 😭